server/routes/linkTokens.js

```diff
@@ -45,20 +43,28 @@ router.post(
         products,
         country_codes: ['US'],
         language: 'en',
-        webhook: httpTunnel.public_url + '/services/webhook',
         access_token: accessToken,
       };
       // If user has entered a redirect uri in the .env file
       if (redirect_uri.indexOf('http') === 0) {
         linkTokenParams.redirect_uri = redirect_uri;
       }
+      // If user has entered a webhook in the .env file
+      if (PLAID_WEBHOOK_URL.indexOf('http') === 0) {
+        linkTokenParams.webhook = PLAID_WEBHOOK_URL;
+      }
```

Comment:
I mentioned this to Todd in person, but it strikes me that if we want to support specifying a PLAID_WEBHOOK_URL we'd need to update .env.template to have instructions on doing this. I am a big fan of removing the ngrok dependency, though, especially as ngrok just broke us again (this is at least the second time that a breaking change to ngrok has broken pattern).